web/build.gradle

```diff
@@ -8,7 +8,7 @@ def springfoxVersion = '3.0.0'
 def springfoxUiVersion = '2.9.2'
 
 group = 'com.jos.dem.jmailer'
-version = '1.0.8'
+version = '1.0.8.2'
```

Comment:
Why it is not 1.0.9? Don't we want to use semver?

Reply:
Semantic versioning is the most popular versioning type. 
I would not force you to use that, just wanted to know what was the reason behind w.x.y.z.
Would be great to mention that versioning system in README

Reply:
I agree; I will add this convention to the wiki page.